src/main/connector/system/LinuxProcess/LinuxProcess.yaml

```diff
@@ -0,0 +1,53 @@
+extends:
+- ../System/System
+connector:
+  displayName: LinuxProcess
+  platforms: Any platform with LinuxOS
+  reliesOn: Linux OsCommands
+  information: Gives process specific information and metrics
+  detection:
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+    - linux
+    criteria:
+    - type: commandLine
+      commandLine: /usr/bin/uname -o || /bin/uname -o
```

Comment:
Instead of running `uname`, we probably should run `ps` or `which ps` (or something like that), so that the connector works only if `ps` works on the system (which is the utility we use here).